projects/common/src/time/interval-duration.service.ts

```diff
@@ -69,7 +69,7 @@ export class IntervalDurationService {
     return durations[0];
   }
 
-  private getAvailableIntervals(timeRange: TimeRange, minDataPoints: number, maxDataPoints: number): TimeDuration[] {
+  public getAvailableIntervals(timeRange: TimeRange, minDataPoints: number, maxDataPoints: number): TimeDuration[] {
```

Comment:
As you mentioned, let's use `getAutoDuration()` instead for our use case. We can remove this change for now

Reply:
yep removed this change